pocket_coffea/lib/jets.py

```diff
@@ -363,6 +363,96 @@ def jet_selection(events, jet_type, params, year, leptons_collection="", jet_tag
     return jets[mask_good_jets], mask_good_jets
 
 
+def jet_selection_nanoaodv12(events, jet_type, params, year, leptons_collection="", jet_tagger=""):
+
+    jets = events[jet_type]
+    cuts = params.object_preselection[jet_type]
+
+    # Bug fix for NanoAOD v12 https://gitlab.cern.ch/cms-jetmet/coordination/coordination/-/issues/117
+    
+    mask_presel = (
+        (jets.pt > cuts["pt"])
+        & (np.abs(jets.eta) < cuts["eta"])
+    )
+    passJetIdTight = ak.zeros_like(jets.pt, dtype=bool)
+    passJetIdTightLepVeto = ak.zeros_like(jets.pt, dtype=bool)
+    
+    passJetIdTight = ak.where((np.abs(jets.eta) <= 2.7), (jets.jetId & (1 << 1))>0, passJetIdTight)
```

Comment:
Can we also include the cases where the requested jetid in the parameters is looser ? in order to simulate the same behaviour of the `jet_selection` standard function